framework/src/modules/interoperability/schemas.ts

```diff
@@ -596,25 +596,13 @@ export const getChainAccountRequestSchema = {
 	required: ['chainID'],
 	properties: {
 		chainID: {
-			dataType: 'bytes',
-			fieldNumber: 1,
```

Comment:
Just want to clarify if `fieldNumber` is not needed if there's only one field

Reply:
I think fieldNumber is required in codec but need to double check 🤔